lib/LLVMIRCodeGen/libobj/Readme.md

```diff
@@ -0,0 +1,19 @@
+# Archiving additional objects within the bundle object
+
+All the object files (.o) stored in this directory will be automatically
```

Comment:
LLVMIRCodeGen is a reusable library component used by different LLVMBackend-derived backends. Those backends may generate code for different targets and architectures. Thus we putting target-specific objects files into such a shared directory does not make a lot of sense, IMHO. If at all, they should be placed in the backend specific directories.

Reply:
Yeah maybe we could change the location of this directory. Since there is a strict convention about the location of the backends, we could choose to select all the files located in the `libobj` sub-directory from each backend directory (if exists), for example `glow/lib/Backends/CPU/libobj`. Although I would also keep this directory for resources which could be common to all backends (e.g. object files which contain data only but no code).